static/shared/js/typeahead.js

```diff
@@ -98,13 +98,21 @@ export function clean_query_lowercase(query) {
     return query;
 }
 
+export const parseUnicodeEmojiCode = (code) =>
```

Comment:
We use `_` in variable names in the web app, and should probably preserve that convention here.

Reply:
Fixed.